cognite/neat/_client/_api/data_modeling_loaders.py

```diff
@@ -561,6 +595,28 @@ def get_id(cls, item: View | ViewApply | ViewId | dict) -> ViewId:
             return ViewId.load(item)
         return item
 
+    @classmethod
+    def merge(cls, local: ViewApply, remote: View) -> ViewApply:
+        if local.as_id() != remote.as_id():
+            raise ValueError(f"Cannot merge views with different IDs: {local.as_id()} and {remote.as_id()}")
+        remote_write = remote.as_write()
+        existing_properties = remote_write.properties or {}
+        merged_properties = {**existing_properties, **(local.properties or {})}
```

Comment:
any chance one of these dict merge could fail at some point?

Reply:
Same argument as above.